07-performance.Rmd

```diff
@@ -101,8 +116,7 @@ system.time({
 ### Reversing elements {-}
 
 The `rev` function provides a reversed version of its argument. If you wish to sort in
-increasing order, then use the more efficient `sort(x, decreasing=TRUE)` instead
-of `rev(sort(x))`.
+increasing order, `sort(x, decreasing=TRUE)` is marginally faster (around 10%) than `rev(sort(x))`.
```

Comment:
Not following our own style here, tut tut - should be `decreasing = TRUE`, with spaces. Also the brackets should be shifted back one word. In total I think the line should be:
```
increasing order, `sort(x, decreasing = TRUE)` is marginally (around 10%) faster than `rev(sort(x))`.
```